### PR TITLE
Added gitattributes files

### PR DESCRIPTION
**Pull request by ihormedvid**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
[the remark this answers is not in the document]
Comment (ihormedvid) [reply pom.xml]: fixed

File: src/main/java/com/javatrainee/App.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+/**
+ * Hello world!
+ *
+ */
+public class App 
+{
+    public static void main( String[] args )
+    {
+
+        System.out.println( "Hello World!" );
```
[the remark this answers is not in the document]
Comment (ihormedvid) [reply src/main/java/com/javatrainee/App.java]: fixed

File: src/test/java/com/javatrainee/AppTest.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+ */
+public class AppTest 
+{
+    /**
+     * Rigorous Test :-)
+     */
+    @Test
+    public void shouldAnswerWithTrue()
+    {
+        assertTrue( true );
```
[the remark this answers is not in the document]
Comment (ihormedvid) [reply src/test/java/com/javatrainee/AppTest.java]: fixed

**Comment by yurii-polishchuk** [comment]
@ihormedvid please fix conflicts

**Pull request merged**